functions/openhabian.bash

```diff
@@ -298,14 +291,6 @@ migrate_installation() {
     fi
```

Comment:
@mstormi Should we even need to keep this now?

Reply:
no lets remove

Reply:
Sounds good, I like that plan. Then it removes the responsibility from us but allows users who really want to to do it on their own using tools that will probably work better because its hard for us to account for everything in an automated migration.